src/ansys/additive/core/simulation_task_manager.py

```diff
@@ -78,6 +80,12 @@ def wait_all(self, progress_handler: IProgressHandler | None = None) -> None:
         for t in self._tasks:
             t.wait(progress_handler=progress_handler)
 
+    def cancel_all(self) -> None:
+        """Cancel all simulations belonging to this simulation task manager."""
```

Comment:
@deleon117 Is this comment intended for users to see? Have we introduced the term task manager? (I just did a search in PyAdditive docs and didn't see it.)

Reply:
@julieatansys